endpoints.json

```diff
@@ -20,15 +20,78 @@
     "exampleResponse": {
       "reviews": [
         {
+          "review_id": 3,
           "title": "One Night Ultimate Werewolf",
           "designer": "Akihisa Okui",
           "owner": "happyamy2016",
+          "review_body": "We couldn't find the werewolf!",
           "review_img_url": "https://images.pexels.com/photos/5350049/pexels-photo-5350049.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260",
           "category": "hidden-roles",
           "created_at": 1610964101251,
-          "votes": 5
+          "votes": 5,
+          "comment_count": 3
         }
       ]
     }
+  },
+  "GET /api/reviews/:review_id": {
+    "description": "serves a review object requested in the endpoint",
+    "queries": [],
+    "exampleResponse": {
+      "review": {
+        "review_id": 3,
+        "title": "One Night Ultimate Werewolf",
+        "designer": "Akihisa Okui",
+        "owner": "happyamy2016",
+        "review_body": "We couldn't find the werewolf!",
+        "review_img_url": "https://images.pexels.com/photos/5350049/pexels-photo-5350049.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260",
+        "category": "hidden-roles",
+        "created_at": 1610964101251,
+        "votes": 5,
+        "comment_count": 3
+      }
+    }
+  },
+  "GET /api/reviews/:review_id/comments": {
+    "description": "serves an array of all the comments associated with the end point's review_id",
+    "queries": [],
+    "exampleResponse": {
+      "comments": {
+        "comment_id": 1,
+        "body": "My dog loved this game too!",
+        "votes": 13,
+        "author": "mallionaire",
+        "review_id": 3,
+        "created_at": "2021-01-18T10:01:41.251Z"
+      }
+    }
+  },
+  "POST /api/reviews/:review_id/comments": {
+    "description": "takes a request of accepted usernames and a comment body to insert into database, serves the string comment",
+    "queries": [],
+    "exampleResponse": {
+      "postedComment": "First"
+    }
+  },
+  "PATCH /api/reviews/:review_id": {
+    "description": "takes a number, updates the votes for the review_id, serves the updated row",
+    "queries": [],
+    "exampleResponse": {
+      "updatedReview": {
+        "review_id": 2,
+        "title": "Jenga",
+        "designer": "Leslie Scott",
+        "owner": "philippaclaire9",
+        "review_img_url": "https://images.pexels.com/photos/4473494/pexels-photo-4473494.jpeg?w=700&h=700",
+        "review_body": "Fiddly fun for all the family",
+        "created_at": "2021-01-18T10:01:41.251Z",
+        "votes": 6
+      }
+    }
+  },
```

Comment:
These two end points will need an exampleBody key with an example of the object expect to be sent i.e. for the post request it would need to have a username and body.